scenarios/sbom.md

```diff
@@ -0,0 +1,4 @@
+# The SBOM Scenario
```

Comment:
SBOM + VDR + NIST self attestation should be addressed in a single scenario. This way we are addressing how SCITT serves the EO needs but that is **only one of the multiple scenarios** that SCITT addresses (other scenarios that can be captured are fuel pump, HW/microelectronics, firmware,..)

Reply:
@kkarunakaran89 , I agree. These docs will support multiple use-cases, with establishing context for how we're thinking about storing, distributing, etc. 
Just so I don't lose your point, was there an additional action item to take in the PR, or just that we need more use-cases, including incorporating the output from: https://github.com/ietf-scitt/draft-birkholz-scitt-software-supply-chain-use-cases/

Reply:
@SteveLasker , no more concerns! Initially when I read the doc structure, I misunderstood the intent of capturing SBOM. I thought you were mentioning that as an independent scenario but it's captured as a component within the SW scenario (SW workflow) under the 'Scenario SCITT enables' section, and that is appropriate.
On a separate note, the long term plan is to eventually build a consolidated use case document (HW, SW, FW..) but that will have to wait until post IETF115

Reply:
I added an eNotary doc from the hack doc that was initially reviewed. The eNotary doc needs more editing, but at least we have it posted for additional PRs and comments